packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -236,15 +236,22 @@ export default function Scroller({
         // scrolled yet, we should still be locked to the anchor post, so this
         // will re-scroll on subsequent layouts as well as the first.
         if (!userHasScrolledRef.current) {
+          // If we're in a gallery channel, we need to adjust the index to account
+          // for the empty post we added after the first unread post.
+          const galleryAdjustedIndex =
+            channelType === 'gallery' && firstUnreadId !== null
```

Comment:
should this be conditional on the same `% 2` check we're using to insert the empty post?

Reply:
Good point

Reply:
Just checked on this, works fine without that check.